core/domain/user_services.py

```diff
@@ -551,3 +560,28 @@ def get_user_impact_score(user_id):
         return model.impact_score
     else:
         return 0
+
+def fetch_gravatar(email):
+    """Returns the gravatar corresponding to the user's email,
+
+    or an default image if gravtar is not receieved
+    """
+    default_identicon_data_url = feconf.DEFAULT_IDENTICON_DATA_URL
```

Comment:
No need to re-declare this. Just refer to feconf.DEFAULT_IDENTICON_DATA_URL where you need it.